FlorenceBML/BMLacq306/BMLacq306.xml

```diff
@@ -223,8 +223,48 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <msPart xml:id="p1"><msIdentifier/><msContents><summary/>
                         <msItem xml:id="p1_i1">
                             <locus from="1ra" to="56rb"/> 
-                            <title type="incomplete" ref="LIT0000">Dəggwā</title>
+                            <title type="incomplete" ref="LIT3178Deggwa">Dəggwā for the whole year in an abridged version</title>
+                            <note>Several folia are lost. Does not contain <title ref="LIT2361somade"/> and 
+                                <title ref="LIT0000">alleluiatic table</title>, ends suddenly with the Transit of Mary. Probably part of a 
+                                <title type="incomplete" ref="LIT1979Mashaf"/></note>
                             <incipit xml:lang="gez">ምልጣን፡ ዘሰርክ፡ በ፩፡ ብፁዕ፡ አንተ፡ ዮሐንስ፡ ዘሀለወከ፡ ታእምር፡ ወተሐውር፡ ቅድመ፡ እግዚአብሔር፡ ጽሊ፡ በእንቲአነ።</incipit>
+                            <explicit xml:lang="gez">ዘፍልሰታ። ዮም፡ ፍሥሐ፡ ኮነ፡ በእንተ፡ ፍልሰታ፡ ለማርያም፡ እምድር፡ እስከ፡ አርያም፡ ዕፁብ፡ ግብር፡ ተገብረ፡ ለባሕቲታ፡ አማን፡ መንክር፡
+                                ሰብሐተ፡ ዕርገታ። ተሠፍሐ፡ እግዝእትየ፡ ለአብርሃም፡ ገራኅቱ፡ ለሙሴ፡ እኅቱ፡ ለዳዊት፡ ወቦቱ፡ ዕንቍ፡ ክቡር፡ ውስቴታ፡ የሐቱ፡ ፄና፡ አልባሲሃ፡ አፈዋት፡ ሎቱ። ዕዝል፡ እስመ፡
+                                ኪያኪ፡ ኃርየ፡ ለታዕካሁ፡ ከመ፡ ትኩኒዮ፡ ማኅደረ፡ ሃሌ፡ ሉ፡ ሃ፡ ሃሌ፡ ማርያምሰ፡ እምነ፡ ወእሙ፡ ለእግዚእነ፡ ሰአሊ፡ በእንቲአነ። ምልጣን፡ እስመ፡ ኪያኪ፡ በል። በ፫እኅትነ፡ 
+                                ይብልዋ፡ ወይኬልልዋ፡ ነያ፡ ሐዳለዩ፡ ጣዕዋ፡ ፀቃውዕ፡ ይውሕዝ፡ እምከናፍሪሃ፡ እ፡ ከመ፡ ቅርፍተ፡ ሮማን፡ መገርሒሃ፡ እ፡ ልዑል፡ ሠምራ፡ ዳዊት፡ ዘመራ፡ በቤተ፡ መቅደስ፡
+                                ተወክፍዋ። ዝማሬ፡ እስመ፡ ኪ</explicit>
+                            <decoNote xml:id="d1" type="ornamentation">Small ornament in the text, that mark the beginning of a new section at 
```

Comment:
decoNote is part of physDesc

Reply:
Ok. I put it to the physDesc (following the handDesc) and nested it in the tag decoDesc.